conbench/templates/c-benchmark-results-for-case.html

```diff
@@ -137,53 +143,71 @@ <h4 class="mb-3">
             sorted: 0,
             show: true,
             spanGaps: false,
-            stroke: 'rgb(87, 125, 134)',
-            width: 0.5,
+            stroke: "#812570", // dark magenta, main line color
+            width: 0.6,
             // I have been taking greenish colors from this palette:
             // https://colorhunt.co/palette/b9eddd87cbb9569daa577d86 We can
             // think about greenifying some graphs, and reddifying others,
             // based on their properties.
-            fill: "rgba(135, 203, 185, 0.4)",
+            fill: "#38BDD122", // skyblue, transp
             //fill: "rgba(255, 0, 0, 0.3)",
             //dash: [10, 5],
             scale: "y",
             points: {
-                // Without `show: true`, points do not always show.
+              // Without `show: true`, points do not always show.
               show: true,
               size: 5,
               fill: 'rgba(87, 125, 134, 0.7)'
             }
           }
         ],
         axes: [
-          {},
+          {
+            label: "benchmark time",
```

Comment:
Great feedback. 

First I had "benchmark start time", because that would be precise/consistent with how it's specified.

Then I found it a little bulky. I agree that "benchmark time" is too ambiguous.

 I think `date` might be good together with maybe a tool tip that explains what exactly this is.